lib/v2/nber/index.js

```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+const { parseDate } = require('@/utils/parse-date');
+
+async function getData(url) {
+    const response = await got(url).json();
+    return response.results;
+}
+
+module.exports = async (ctx) => {
+    const url = 'https://www.nber.org/api/v1/working_page_listing/contentType/working_paper/_/_/search';
+    const baseUrl = 'https://www.nber.org';
+    const data = ctx.cache.tryGet(url, async () => {
+        const response = await getData(url);
+        return response;
+    });
+    const items = await Promise.all(
+        data
+            .filter((article) => ctx.path === '/papers' || article.newthisweek)
+            .map((article) => {
+                const link = `${baseUrl}${article.url}`;
+                const item = ctx.cache.tryGet(link, async () => {
```

Comment:
However, you don't need to `await` for it if it's returned inside an async function(`module.exports = async (ctx) => {...`).
```suggestion
                return ctx.cache.tryGet(link, async () => {
```